#144: Added documentation regarding installation of starter scripts and added contributing.md (#149)
exasol/script-languages-container-tool: contributing.md

# Contributing to the Script Language Container Tool

## :tada: :+1: First off, thanks for taking the time to contribute! :tada: :+1:

The following is a set of guidelines for contributing to the Script Language Container Tool. 
These are mostly guidelines, not rules. Use your best judgment, and feel free 
to propose changes to this document in a pull request. 
Furthermore, they contain some information which might help you during development.

## How can you contribute?

You can contribute to this project at several levels:

> - In any case we suggest you to create a new Github issue, be it for bug reports or new features, before implementing any change. Thus, we can give you feedback soon. 

Reviewer: ```suggestion
- In any case, we recommend creating a new Github issue for bug reports or new features before implementing any changes. Thus, we can give you feedback soon. 
```
